tox.ini

```diff
@@ -1,44 +1,88 @@
+# This file is a local convenience. It is not a substitute for the full CI
+# suite, and does not cover the full range of Python versions for Mitogen.
+
+# I use this on Ubuntu 20.04, with the following additions
```

Comment:
I think it would be cool to have a `Dockerfile` of the `tox` env, so that users could run the tox tests inside of that locally if they want to. WDYT? It would also serve as a sandbox to install anything that Mitogen's tests install in a separate env than the user's host system.

Reply:
Possibly, there are some tox-but-with-docker projects on PyPI. I'll dig out the links.

Reply:
https://github.com/mitogen-hq/mitogen/pull/607 has discussion of tox × docker